pkg/blobcache/blobcache.go

```diff
@@ -416,6 +416,18 @@ func (s *blobCacheDestination) HasThreadSafePutBlob() bool {
 	return s.destination.HasThreadSafePutBlob()
 }
 
+func (s *blobCacheDestination) SupportsBlobLocks() bool {
+	return false
```

Comment:
(Note that the cache object can wrap a `storageImageDestination` in `Commit`. I don’t quite think that it matters — we are only actually creating some new layers, so there should not be any concurrent downloads of that.)